utils/R/ensemble.R

```diff
@@ -62,8 +62,6 @@ read.ensemble.output <- function(ensemble.size, pecandir, outdir,
 ##' @return matrix of random samples from trait distributions
```

Comment:
remove 'random' from definition/ title. 

for method parameter list options preferably with guidance for choosing a method